modules/ebs_snapshot_explorer/main.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python3
+"""Module for ebs_snapshot_explorer"""
+from copy import deepcopy
+import json
+from pathlib import Path
+
+from botocore.exceptions import ClientError
+
+from . import parser
+from . import module_info
+
+
+def input_helper(input):
+    """Helper function that loops until a successful response is given"""
+    prompt = '    Load next set of volumes? (y/n) '
```

Comment:
Probably could just put this into the input, since this is the only place it is being used